samples/python/onedim/species_sensitivity.py

```diff
@@ -0,0 +1,106 @@
+r"""
+Species sensitivity analysis
+========================================
+
+In this example we simulate an impinging jet flame, premixed ammonia/hydrogen-air flame,
+calculate the sensitivity of N2O to the A-factor reaction rate constants.
+The input has been set for a given inlet velocity and flame wall temperature.
+
+Requires: cantera >= 3.0.0, pandas
+
+.. tags:: Python, combustion, 1D flow, species reaction, premixed flame,
+          sensitivity analysis, plotting
+
+
+Analysis on ammonia flame is undertaken using the Nakamura mechanism:
+[1] H. Nakamura, S. Hasegawa, T. Tezuka, Kinetic modeling of ammonia/air weak flames in a micro flow reactor with a
+controlled temperature profile, Combust. Flame 185 (2017) 16-27.
+[2] H. Nakamura, S. Hasegawa, T. Tezuka, available from “http://www.ifs.tohoku.ac.jp/enerdyn/en/result/mechanism.html”
+
+Simulation is undertaken to recreate the N2O sensitivity analysis by Hayakawa et al.:
+[3] A. Hayakawa, M. Hayashi, M. Kovaleva, G.J. Gotama, E.C. Okafor, S. Colson, S. Mashruk, A. Valera-Medina, T. Kudo,
+ H. Kobayashi, Experimental and numerical study of product gas and N2O emission characteristics of ammonia/hydrogen/air premixed laminar flames stabilized in a stagnation flow, Proc. Combust. Inst. 39 (2023) 1625–1633. https://doi.org/https://doi.org/10.1016/j.proci.2022.08.124.
+"""
+
+import cantera as ct
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+# Define the gas mixture
+gas = ct.Solution("nakamura.yaml")  # Use the Nakamura mechanism for ammonia combustion blends
```

Comment:
```suggestion
gas = ct.Solution("example_data/nakamura-2019.yaml")  # Use the Nakamura mechanism for ammonia combustion blends
```
The path would have to be updated to reflect the file location.

Reply:
@ischoegl Thank you - working on this now, please bear with me.

Reply:
Hi @ischoegl 
I've made the changes requested:

Branch now updated to main. 
Nakamura.yaml has been added as a mech to the example data folder
The sensitivity function uses index instead of location. 

The CI runner has triggered a new request - does this need to be re-run. If so, please approve.

Reply:
@ischoegl CI runner requests should now (hopefully) all be fixed apart from one Sphinx documentation error. This is triggered by a "/' located in a reference in the Nakamura.yaml file. I'm having issues navigating and editing this file now as it's in the submodule. Any advice would be appreciated.